websites/demo-react-image-next/src/app/providers.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import { MediaQueryProvider } from "@jsxcss/emotion";
+import { Suspensive, SuspensiveProvider } from "@suspensive/react";
+import { PropsWithChildren } from "react";
```

Comment:
In the case of `type`, I would like to specify it, but it seems that **Lint** is starting to rely on `@suspensive/react`. How to solve this?